project_profitability_fields/views/project_project.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="project_custom_fields_form_view" model="ir.ui.view">
+        <field name="name">project.project.form</field>
+        <field name="model">project.project</field>
+        <field name="inherit_id" ref="project.edit_project" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='tag_ids']" position="after">

```

Comment:
We should considere some things about this fields rendering:
- Placing them in a `page` of existing `notebook`, rather than main group form.
- Which roles should access this fields to be shown and/or modified? E.g. project managers should only view and edit them, but I'm not sure.

Please ask LR for both questions/doubts